pilot/pkg/networking/core/tracing.go

```diff
@@ -577,8 +576,7 @@ func configureSampling(hcmTracing *hcm.HttpConnectionManager_Tracing, providerPe
 }
 
 func proxyConfigSamplingValue(config *meshconfig.ProxyConfig) float64 {
-	// PILOT_TRACE_SAMPLING
-	sampling := features.TraceSampling
+	sampling := 1.0
```

Comment:
how can this be turned off? `config.Tracing.Sampling != 0.0 {` does not allow it

Reply:
what do you mean turned off? sampling = 0.0?

Reply:
The questions is what is the recommended way to configure trace sampling for pilot. I assumed using the OTel env - https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/ -  but this method is confusing me, shouldn't you remove this method and let OTel do its own config ?